ts/features/itwallet/analytics/index.ts

```diff
@@ -785,13 +786,19 @@ export const trackItwRequest = (ITW_ID_method?: ItwIdMethod) => {
   }
 };
 
-export const trackItwRequestSuccess = (ITW_ID_method?: ItwIdMethod) => {
+export const trackItwRequestSuccess = ({
+  ITW_ID_method,
+  ITW_ID_status
+}: {
+  ITW_ID_method: ItwIdMethod;
+  ITW_ID_status: ItwStatus;
+}) => {
```

Comment:
This is easier to read and it's backward compatible. What do you think?

```suggestion
export const trackItwRequestSuccess = (method?: ItwIdMethod, status?: ItwStatus) => {
```

Reply:
Fixed, thanks!